code_schemes/s01e02.json

```diff
@@ -33,6 +33,62 @@
         "VisibleInCoda": true,
         "Shortcut": "s"
       },
+      {
+        "CodeID": "code-57aeb565",
+        "CodeType": "Normal",
+        "DisplayText": "unclear who is speaking",
+        "NumericValue": 1,
+        "StringValue": "unclear_who_is_speaking",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-51881163",
+        "CodeType": "Normal",
+        "DisplayText": "prayer",
+        "NumericValue": 2,
+        "StringValue": "prayer",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-73db94b0",
+        "CodeType": "Normal",
+        "DisplayText": "follow the advice of the government",
+        "NumericValue": 3,
+        "StringValue": "follow_the_advice_of_the_government",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-9c765886",
+        "CodeType": "Normal",
+        "DisplayText": "denial",
+        "NumericValue": 4,
+        "StringValue": "denial",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-d04669f3",
+        "CodeType": "Normal",
+        "DisplayText": "rumour misinfo",
+        "NumericValue": 5,
+        "StringValue": "rumour_misinfo",
```

Comment:
(over on the ke projects this is rumour/stigma/misinfo)

Reply:
Thanks, @as2388. @lukechurch should we update this theme to match urban-ke - rumour/stigma/misinfo?

Reply:
sure